exploration/seasons.md

```diff
@@ -0,0 +1,161 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.15.2
+  kernelspec:
+    display_name: ds-aa-sahel-drought
+    language: python
+    name: ds-aa-sahel-drought
+---
+
+# ASAP phenology
+
+```python
+%load_ext jupyter_black
+%load_ext autoreload
+%autoreload 2
+```
+
+```python
+import os
+from pathlib import Path
+
+import datetime
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import pandas as pd
+import rioxarray as rxr
+import numpy as np
+import xarray as xr
+from tqdm.notebook import tqdm
+
+from src import utils
+```
+
+```python
+DATA_DIR = Path(os.getenv("AA_DATA_DIR"))
+```
+
+```python
+aoi = utils.load_codab_aoi()
+```
+
+```python
+# utils.process_asap_raw()
+```
+
+```python
+da = utils.load_asap_sos_eos()
+da = da.rio.clip(aoi.geometry, all_touched=True)
+da = da.where(da < 251)
+da = da.assign_attrs({"_FillValue": np.nan})
+# bring all into two years
+da = da.where(da < 37, da - 36)
+# da = da.where(da < 37, da - 36)
+```
+
+```python
+def dekad_to_month(dekad, pos=None):
+    if dekad < 1:
+        return None
+    dekad = np.round(dekad).astype(int)
+    dekad = dekad - 72 if dekad > 72 else dekad
```

Comment:
`dekad = ((np.round(dekad).astype(int) - 1) % 36) + 1` for a one liner.

Reply:
v nice 07fb0ee8a9cb4d969cdaf356e5592c319212c81c